src/fides/api/service/saas_request/override_implementations/zendesk_request_overrides.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Dict, List
+from urllib import parse
+from urllib.parse import urlsplit
+
+import pydash
+
+from fides.api.common_exceptions import FidesopsException
+from fides.api.graph.traversal import TraversalNode
+from fides.api.models.policy import Policy
+from fides.api.models.privacy_request import PrivacyRequest
+from fides.api.schemas.policy import ActionType
+from fides.api.schemas.saas.shared_schemas import HTTPMethod, SaaSRequestParams
+from fides.api.service.connectors.saas.authenticated_client import AuthenticatedClient
+from fides.api.service.saas_request.saas_request_override_factory import (
+    SaaSRequestType,
+    register,
+)
+from fides.api.util.collection_util import Row
+
+## Uplifted from Connectors-logitech
+## See https://github.com/ethyca/connectors-logitech/blob/main/connectors/zendesk/zendesk_request_overrides.py#L95
+
+
+def _check_tickets(tickets: List[Row], policy: Policy) -> None:
+    """
+    Raises an exception if there are any "open" tickets to halt the privacy request.
+    Will only halt if the policy contains an erasure action.
+    """
+    if policy.get_rules_for_action(action_type=ActionType.erasure):
+        for ticket in tickets:
+            ## TODO: Check with Ramp that the Statuses list is the expected one
+            if ticket["status"] in ["new", "open", "pending", "hold", "solved"]:
+                raise FidesopsException("User still has open tickets, halting request")
+
+
+@register("zendesk_logitech_tickets_read", [SaaSRequestType.READ])
+def zendesk_logitech_tickets_read(
```

Comment:
Same here, let's just call it `zendesk_tickets_read`
```suggestion
def zendesk_tickets_read(
```